ZenPacks/zenoss/AWS/dsplugins.py

```
@@ -444,6 +444,13 @@ def inner():
                         else:
                             message = msg_dict.get('Message')
                             summary = msg_dict.get('Subject')
+                        try:
+                            message = json.loads(message)
+                            summary = message.get('notificationType')
+                            message = json.dumps(message, indent=4, sort_keys=True)
+                        except (ValueError,TypeError,AttributeError):
+                            pass
```

Silently dropping the message if it doesn't contain the expected structure is probably not a good move here.  I'd pass the original message through in that case.